File: Modul-2/PRAK203/PRAK203-2310817210010-Harry-Pratama-Yunus.c
#include <stdio.h>

int main (){
    float a,b,i,j,x,y;
    scanf("%f %f %f %f %f %f",&a,&b,&i,&j,&x,&y);
    float c=a-b,z=x+y;
    float k=i/j;
    float Hasil=c*k-z;
    printf("%.3f",Hasil);
    return 0;
}

File: Modul-2/PRAK203/PRAK203-2310817210010-Harry-Pratama-Yunus.py
Angka=input().split()
if len (Angka) == 2:
    a,b=map(float,Angka)
    i,j=map(float,input().split())
    x,y=map(float,input().split())
elif len (Angka) == 6:
    a,b,i,j,x,y=map(float,Angka)
c=a-b
z=x+y
k=i/j
Hasil=c*k-z
print("{:.3f}".format(Hasil))